1_fetch.R

```diff
@@ -308,6 +308,21 @@ p1_targets_list <- list(
      file.path(dirname(p1_natural_baseflow_zip), list.files(path = dirname(p1_natural_baseflow_zip),pattern = "*baseflow.*.csv"))
       },
     format = "file"
+  ),
+  
+  # Load synoptic sampling sites.
+  # Rename columns to match needs of site to reach function
+  tar_target(
+    p1_syn_sites,
+    {
+      if(!('DRB_2021_synoptic_sites_by_transect_20210819.xlsx' %in% 
+           list.files('1_fetch/in/'))){
+        stop('Please add DRB_2021_synoptic_sites_by_transect_20210819.xlsx to 1_fetch/in')
```

Comment:
I really like this additional code to stop the pipeline if the synoptic sites file isn't detected. Thanks for including that!